eng/references.targets

```diff
@@ -70,15 +70,15 @@
 
   <ItemDefinitionGroup>
     <TargetPathWithTargetPlatformMoniker>
-      <IsReferenceAssembly>$(IsReferenceAssembly)</IsReferenceAssembly>
+      <IsReferenceAssemblyProject>$(IsReferenceAssemblyProject)</IsReferenceAssemblyProject>
```

Comment:
This looks like `x = x`.... does it do anything?

Reply:
Yes, it adds the metadata to the item so that it can then be read in the target below.